src/m365/spo/commands/page/page-set.ts

```diff
@@ -52,7 +60,8 @@ class SpoPageSetCommand extends SpoCommand {
         publish: args.options.publish || false,
         publishMessage: typeof args.options.publishMessage !== 'undefined',
         description: typeof args.options.description !== 'undefined',
-        title: typeof args.options.title !== 'undefined'
+        title: typeof args.options.title !== 'undefined',
+        demotefrom: typeof args.options.demoteFrom !== 'undefined'
```

Comment:
Since this is a choice field, we can log the choice to the telemetry.
```suggestion
        demotefrom: args.options.demoteFrom || false,
```

Reply:
Thanks for making me aware of this.